.gitignore

```diff
@@ -64,4 +64,7 @@ docs/_build/
 target/
 
 # IPython Notebooks
-.ipynb_checkpoints/*
+.ipynb_checkpoints
```

Comment:
Actually, I'll just merge this and do that myself. Thanks again for working on the docs! :+1: